src/components/molecules/HamDropDown/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+
+import Styled from 'styled-components';
+import arrowDown from 'assets/images/ham_arrowDown.svg';
+import arrowUp from 'assets/images/ham_arrowUp.svg';
+import Button from '../../atoms/Button/index';
+
+export interface IProps {
+  className?: string;
+  title: string;
+  list: string[];
+}
+
+function HamDropDown({ ...props }: IProps): React.ReactElement {
+  const { title, list } = props;
+  const [opened, setOpened] = useState(false);
```

Comment:
보통 True, False로 분리되는 변수의 경우는 is~를 붙입니다!
```suggestion
  const [isOpened, setIsOpened] = useState(false);
```